examples/simple-app-command.yml

```diff
@@ -0,0 +1,49 @@
+apiVersion: kappctrl.k14s.io/v1alpha1
+kind: App
+metadata:
+  name: simple-app
+  namespace: default
+spec:
+  serviceAccountName: default-ns-sa
+  fetch:
+    - image:
+        subPath: monitoring
+        url: arttii/dummydeploy
+  template:
+  - command: 
+      cmd: "sops"
```

Comment:
Where is the binary coming from? Should this allow (or require) specifying the image to run? Would we implement the other templating logic (ytt, helm, jsonnet,...) using this approach?

Reply:
Well, it should be present on the system the controller in running on, so in the Docker. So if you want to use custom commands like this, you need to build a custom Docker.
I would not implement all templating with this, I think it's better to have a more specific implementation for functions, that we know users need. This one is more or less for the case, when one is not available or does not make a lot of sense it integrate as an inbuilt templating function.